lang/c++/include/avro/Validator.hh

```diff
@@ -35,7 +35,7 @@ public:
     explicit NullValidator(const ValidSchema &) {}
```

Comment:
This file also introduces user-facing sign-ness changes

Reply:
True, I believe the new type is more appropriate here though, as a negative value isn't valid for any of these values... does this deserve a Jira issue?

Reply:
I think yes. Also let's hear from @martin-g as he knows better the release cycle.

But at least this should be properly documented as a breaking change.

Reply:
1.12 is really close to be released!
The JIRA ticket is needed mostly for the CHANGELOG file, so users are aware of the change and can update their code.

Reply:
I created <https://issues.apache.org/jira/browse/AVRO-4019> to track this issue